Dockerfile

```diff
@@ -1,33 +1,37 @@
-FROM postgres:alpine3.18 as supabase-db
-COPY ./tests/Transfer/resources/supabase/backup.tar /docker-entrypoint-initdb.d/backup.tar
-COPY ./tests/Transfer/resources/restore.sh /docker-entrypoint-initdb.d/restore.sh
+FROM supabase/postgres:15.1.0.96 as supabase-db
+COPY ./tests/Transfer/resources/supabase/1_globals.sql /docker-entrypoint-initdb.d/1_globals.sql
+COPY ./tests/Transfer/resources/supabase/2_main.sql /docker-entrypoint-initdb.d/2_main.sql
+RUN rm -rf /docker-entrypoint-initdb.d/migrate.sh
 
 FROM postgres:alpine3.18 as nhost-db
-COPY ./tests/Transfer/resources/nhost/backup.tar /docker-entrypoint-initdb.d/backup.tar
-COPY ./tests/Transfer/resources/restore.sh /docker-entrypoint-initdb.d/restore.sh
+COPY ./tests/Transfer/resources/nhost/1_globals.sql /docker-entrypoint-initdb.d/1_globals.sql
+COPY ./tests/Transfer/resources/nhost/2_main.sql /docker-entrypoint-initdb.d/2_main.sql
 
 # Use my fork of mockoon while waiting for range headers to be merged
-FROM node:14-alpine3.14 as mock-api
+FROM node:20.4-alpine3.17 as mock-api
 WORKDIR /app
-RUN git clone https://github.com/PineappleIOnic/mockoon.git . 
+RUN apk add --no-cache git
+RUN git clone https://github.com/PineappleIOnic/mockoon.git
```

Comment:
Is this added only for testing or will it also be used in prod?

Reply:
Mockoon currently doesn't support range headers, I have a PR in the works to bring this feature over to their repo but currently I don't have the time to write tests for them so for now we are using my fork and will switch once I have more time to finish their tests and merge it into their repo